local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationsPlugin.java

```diff
@@ -221,13 +227,34 @@ public void requestPermissions(PluginCall call) {
         }
     }
 
+    @PluginMethod
+    public void changeExactNotificationSetting(PluginCall call) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+            startActivityForResult(call, new Intent(ACTION_REQUEST_SCHEDULE_EXACT_ALARM), "alarmPermissionsCallback");
+        }
```

Comment:
if `< Build.VERSION_CODES.S` it should call `checkExactNotificationSetting(call);` so it resolves and returns as granted on older versions, as it's now it will hang in previous versions.